### DISCUSSION TITLE
Skipping item - invalid name (Microsoft Naming Convention): ./App/Joplin/.lock

### DISCUSSION THREAD

**Discussion by globdug** [abraunegg/onedrive#2547] [General]
Hello,

I always get this error: `Skipping item - invalid name (Microsoft Naming Convention): ./App/Joplin/.lock` (from OneDrive GUI, but I guess it's a OneDrive issue...).

What kind of file is ".lock"? I always synced that folder with no issues in Windows.

**Comment by abraunegg** [comment]
@globdug 
You get that error as Microsoft advise those files are invalid for OneDrive. Please read: 

https://support.microsoft.com/en-au/office/restrictions-and-limitations-in-onedrive-and-sharepoint-64883a5d-228e-48f5-b3d2-eb39e07630fa#invalidfilefoldernames

> What kind of file is ".lock"? 

You will need to ask the developers of Joplin that and what they use it for

**Reply by globdug** [reply]
Thank you for your reply.
The strange thing is that if the '.lock' file is there, it's because it has been synchronized by Joplin without errors. Moreover, I formatted my pc several time with linux, windows 10, 11 and had to resync onedrive many times without getting errors on Windows. 
I think Windows, sometimes, works in mysterious ways... 😅
